oeps/oep-0001.rst

```diff
@@ -0,0 +1,334 @@
+-------------------------
+
+OEPs are UTF-8 encoded text files using the `reStructuredText`_ format.
+ReStructuredText [8] allows for rich markup that is still quite easy to read,
+but also results in good-looking and functional HTML. OEPs are rendered to HTML
+using Sphinx. An OEP template can be found `in the repo`_.
+
+.. _reStructuredText: http://docutils.sourceforge.net/rst.html
+.. _in the repo: https://github.com/cpennington/open-edx-proposals/blob/master/oep-template.rst
+
+OEP Header Preamble
+-------------------
+Each OEP must begin with an ReST table with meta-data about the OEP. The rows must
+appear in the following order. Rows in italics are optional and
+are described below. All other rows are required.
+
+
++---------------+---------------------------------------------+
+| OEP           | <link to OEP>                               |
++---------------+---------------------------------------------+
+| Title         | <OEP title>                                 |
++---------------+---------------------------------------------+
+| Last-Modified | <date string>                               |
++---------------+---------------------------------------------+
+| Author        | <list of authors' real names and            |
+|               | email addresses>                            |
++---------------+---------------------------------------------+
+| Arbiter       | <OEP Arbiter's real name and email address> |
++---------------+---------------------------------------------+
+| Status        | <Draft | Accepted | Deferred |              |
+|               | Rejected | Withdrawn | Final |              |
+|               | Replaced>                                   |
++---------------+---------------------------------------------+
+| Type          | <Architecture | Best Practice |             |
+|               | Process>                                    |
++---------------+---------------------------------------------+
+|  Created      | <date created on, in YYYY-MM-DD format>     |
++---------------+---------------------------------------------+
+| `Replaces`    | <link to OEP>                               |
++---------------+---------------------------------------------+
+| `Replaced-By` | <link to OEP>                               |
++---------------+---------------------------------------------+
+
+The Author header lists the names, and optionally the email addresses of
+all the authors/owners of the OEP. The format of the Author header value must be
+``Random J. User <address@dom.ain>`` if the email address is included, and just
+``Random J. User`` if the address is not given. If there are multiple authors,
+they should be put in a comma separated list.
+
+The Arbiter field is used to record who the final decision to approve or
+reject an OEP rests with.
+
+The Type header specifies the type of OEP: Architecture, Best Practice or Process.
+
+The Created header records the date that the OEP was assigned a number. It should be
```

Comment:
This implies that OEP numbers are "assigned".  However, earlier, it states that each Author can "claim" their own number.   Are there also guidelines on which numbers can be claimed (always incremental, etc)?